File: Exercicios-Beecrowd/C#/1013.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Teste
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string[] lista= Console.ReadLine().Split();
            int a = Convert.ToInt32(lista[0]);
            int b = Convert.ToInt32(lista[1]);
            int c = Convert.ToInt32(lista[2]);

            if (a >= b && a >= c)
            {
                Console.WriteLine("{0} eh o maior", a);
            }
           else if (b >= a && b >= c)
            {
                Console.WriteLine("{0} eh o maior", b);
            }
            else
            {
                Console.WriteLine("{0} eh o maior", c);

            }

        }
    }
}

File: Exercicios-Beecrowd/Kotlin/1013.kt
import java.util.*

fun main(args: Array<String>) {

    val valor = readLine().toString().split(" ")
    val a=valor[0].toInt()
    val b=valor[1].toInt()
    val c=valor[2].toInt()
    if (a>=b && a>=c){
        println("${a} eh o maior")
    }
    else if (b>=a && b>=c){
        println("${b} eh o maior")

    }
    else{
        println("${c} eh o maior")
    }

	
}